admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -123,12 +123,25 @@ Only those object classes:
   object classes.
 
 Only from those groups:
-  If your LDAP server supports the ``member-of-overlay`` in LDAP filters, you can define that only users from one or more certain groups are allowed to appear in user listings in ownCloud.
+  If your LDAP server supports the ``member-of-overlay`` in LDAP filters, you
```

Comment:
The original 1st commit changed ``member-of-overlay`` to ``memberof-overlay``
But the 3rd commit has reversed that ???

Reply:
Not intentionally. I'll put that back.